overlord/state/notices.go

```diff
@@ -243,7 +259,12 @@ func (s *State) AddNotice(userID *uint32, noticeType NoticeType, key string, opt
 
 	now := options.Time
 	if now.IsZero() {
-		now = time.Now()
+		now = getTimeNow()
+		// ensure that two notices never have the same sent time
+		if compareDates(now, s.noticeLastDate) <= 0 {
```

Comment:
do we need the time compare helper? wouldn't the check below be sufficient?
```suggestion
		if !now.After(s.noticeLastDate) {
```